12/hw3.py

```diff
@@ -0,0 +1,12 @@
+class HighlightBlock:
+    def __enter__(self):
+        print("==========")
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print("==========")
+        if exc_val is not None:
+            print(f"Error: {exc_val}")
+        return False
```

Comment:
Your context manager shouldn't raise an exception. Replace here `False` with `True` to make it works

Reply:
corrected